### PR TITLE
c9.io

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

File: src/chrome/content/rules/Cloud9.xml [hunk 1]
```diff
@@ -11,6 +11,7 @@
     <target host="c9.io" />
 	<target host="docs.c9.io" />
 	<target host="www.c9.io" />
+	<target host="status.c9.io" />
```
Comment (J0WI) [inline src/chrome/content/rules/Cloud9.xml]: c9.io
community.c9.io
apidoc.c9.io
ide.c9.io
get.c9.io
ace.c9.io

**Comment by J0WI** [comment]
@galeksandrp please have a look at our guide, where we documented how to find subdomains: https://github.com/EFForg/https-everywhere/blob/master/ruleset-style.md

**Comment by galeksandrp** [comment]
c9.io have a user-created domains. Do we need a wildcard?

**Comment by galeksandrp** [comment]
Added, thanks!

**Pull request merged**